network-api/networkapi/mozfest/models.py

```diff
@@ -82,6 +83,18 @@ def get_context(self, request, bypass_menu_buildstep=False):
 
 
 class MozfestHomepage(MozfestPrimaryPage):
+    cta_button_label = models.CharField(
```

Comment:
Why are we adding this to the Homepage rather than using [Base settings](https://docs.wagtail.io/en/v2.5.1/reference/contrib/settings.html#defining-settings)?

Reply:
No idea: that link goes nowhere?

Reply:
investigation planned in https://github.com/mozilla/foundation.mozilla.org/issues/3477